package.json

```diff
@@ -51,7 +51,8 @@
     "redux-thunk": "^2.2.0",
     "topojson-client": "^3.0.0",
     "velocity-react": "^1.3.3",
-    "whatwg-fetch": "^2.0.3"
+    "whatwg-fetch": "^2.0.3",
+    "globalfishingwatch-convert": "git+https://github.com/GlobalFishingWatch/map-convert.git"
```

Comment:
not sure if this belongs here

Reply:
Thanks @tiagojsag you are right, that artifact is used in the development branch, not in master. Removed